live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/ports/repository/api/DiscoveryApi.java

```diff
@@ -0,0 +1,31 @@
+/*-
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.live_ingester.domain.ports.repository.api;
+
+public interface DiscoveryApi
+{
+    String getRepositoryVersion();
```

Comment:
I would get rid of this interface tbh, it is not used in `domain` module anyway

Reply:
Good point - I'll move the interface to adapters (it's useful to have it at all so that integration tests configs are easier to mock)

Reply:
Interface removed